### PR TITLE
Updating BeatAML to retrieve the full datset.

### PR DESCRIPTION
**Pull request by jjacobson95**

### CODE REVIEW

**Comment by jjacobson95** [comment]
This branch should be ready to merge with main.

File: build/beatAML/GetBeatAML.py [hunk 1]
```diff
@@ -115,24 +83,80 @@ def modify_patient_file(file_path):
...
+    prot_samples = prot_samples[["labId","specificDxAtInclusion","specimenType"]]
+    prot_samples.rename(columns={"labId": "other_id"}, inplace=True)
+    prot_samples.rename(columns={"specificDxAtInclusion": "other_names"}, inplace=True)
+    prot_samples.rename(columns={"specimenType": "common_name"}, inplace=True)
+    prot_samples["cancer_type"] = "ACUTE MYELOID LEUKAEMIA"
+    prot_samples["model_type"] = "ex vivo"
+    prot_samples["other_id_source"] = "beatAML"    
+    
+    all_samples = pd.concat([prot_samples, full_samples])
+    maxval = max(pd.read_csv('https://raw.githubusercontent.com/PNNL-CompBio/candleDataProcessing/main/build/hcmi/samples.csv').improve_sample_id)
```
Comment (sgosline) [inline build/beatAML/GetBeatAML.py:137]: remove this, please only work locally, as the files should all build in the same place for now, and not require uploading to github.

**Review by sgosline — COMMENTED** [review]
fix that line and we should be good.

File: build/beatAML/GetBeatAML.py (hunk 1 above)
Comment (jjacobson95) [reply build/beatAML/GetBeatAML.py:137]: Just that last line right?  Should I change all instances where a file is downloaded from GitHub? (I’ll have it pull from the CODR repo instead)

Comment (jjacobson95) [reply build/beatAML/GetBeatAML.py:137]: Sorry, I mis-remembered something on this. This is the only instance where it pulls from our Github. And I'll change it to CODR.

Comment (sgosline) [reply build/beatAML/GetBeatAML.py:137]: yeah, i fixed up all the other scripts, this is the only one left.

**Comment by jjacobson95** [comment]
HCMI path changed to local. Should we do something with the [depmap drug path](https://figshare.com/account/projects/170775/articles/24545596?file=43112314) too?

**Comment by sgosline** [comment]
Yes, we should! At least for now.

**Pull request merged**